### ISSUE TITLE
In-Game: Emitter ind found in particle_get_info is a number, not a ref

### ISSUE THREAD

**Issue by KormexGit** [YoYoGames/GameMaker-Bugs#7860]
### Description
If you create a particle using part_emitter_create, you will get something like `ref particle emitter 0`. But, if you use particle_get_info to retrieve the newly added emitter ind found inside,  you will get a number like 0.

### Steps To Reproduce
```
system = part_system_create(ParticleSystem1);
var info = particle_get_info(system);

show_message("Emitter created with part_emitter_create: " + string(part_emitter_create(system))); //shows ref particle_emitter 1
show_message("Emitter found in particle_get_info: " + string(info.emitters[0].ind)); //shows 0
```

### Which version of GameMaker are you reporting this issue for?
IDE v2024.8.1.171 Runtime v2024.8.1.218

### Which operating system(s) are you seeing the problem on?
Windows 10.0.22631.0

### Which platform(s) are you seeing the problem on?
Windows

[2bbb35ec-8027-4af6-a5c7-257df8e842c8](https://api.gamemaker.io/api/github/downloads/2bbb35ec-8027-4af6-a5c7-257df8e842c8)

**Comment by kleof** [comment]
May be related:
```gml
part_system1 = part_system_create_layer("Instances", false, ps_asset);
part_system_position(part_system1, room_width/2, room_height/2);

var info = particle_get_info(ps_asset);
emitter1 = info.emitters[0].ind;
part_type1 = info.emitters[0].parttype.ind;

part_emitter_enable(part_system1, emitter1, true);
part_emitter_burst(part_system1, emitter1, part_type1, 50);
```

`ps_asset` has a bunch of emitters set to Burst Mode that are disabled in the editor.
Code above enables first one and bursts it.
It works as expected, unless the _last_ emitter on the list is set to Stream Mode - then it enables that last emitter, making it stream, and bursts the particles of _first_ emitter.
(if the stream mode emitter is anywhere else on the list, this does not occur)

**Comment by YYBartT** [comment]
Updated the description of the types in the manual to "Particle Emitter" and "Particle Type".

**Comment by Emc1923** [comment]
Verified fixed in  IDE v2024.1300.0.715  Runtime v2024.1300.0.728.
